### PR TITLE
Move the test from the testcafe repo (close #53)

### LINKED ISSUE #53: Move the test from the testcafe repo
I've found the regression test in the testcafe repo that relatives to the testcafe-hammerhead, move it.

```js
test('B238956 - Event sandbox: svg specific native methods not memorized', function () {
        var $svg       = $('<svg xmlns="http://www.w3.org/2000/svg" version="1.1"></svg>')
            .addClass(TEST_ELEMENT_CLASS)
            .width(500)
            .height(300)
            .appendTo('body'),
        clickCount = 0;

    $svg.click(function () {
        clickCount++;
    });

    $svg[0].click();
    equal(clickCount, 1);
});

```

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by miherlosev** [comment]
Test name is not clear.
I think we need rename to 'svg element click event handler must be overriden correctrly (B238956)

**Comment by inikulin** [comment]
@miherlosev Agrred
\r-

**Comment by inikulin** [comment]
@LavrovArtem Check new test name with @MargaritaLoseva BTW

**Comment by inikulin** [comment]
@LavrovArtem Check new test name with @MargaritaLoseva BTW

File: test/client/fixtures/sandbox/event/event-test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-var Browser        = Hammerhead.get('./utils/browser');
-var NativeMethods  = Hammerhead.get('./sandbox/native-methods');
-var Const          = Hammerhead.get('../const');
+var Browser       = Hammerhead.get('./utils/browser');
+var NativeMethods = Hammerhead.get('./sandbox/native-methods');
+var Const         = Hammerhead.get('../const');
```
Comment (churkin) [inline test/client/fixtures/sandbox/event/event-test.js]: browser --> browserUtils
NativeMethods --> nativeMethods
Const --> CONST

**Comment by LavrovArtem** [comment]
FPR

**Comment by churkin** [comment]
lgtm

**Pull request merged**